java/generate-offline-map-with-local-basemap/README.md

```diff
@@ -20,28 +20,31 @@ The author of a web map can support the use of basemaps which are already on a d
 
 ## How to use the sample
 
-1. Click on the "Generate offline map" button.
-1. You will be prompted to choose whether you wish to use the "naperville_imagery.tpk" basemap which is already on the device or download the online basemap.
-1. If you choose to download the online basemap, the offline map will be generated with the same (topographic) basemap as the online web map. To download the Esri basemap, you may be prompted to sign in to ArcGIS.com.
-1. If you choose to use the basemap from the device, the offline map will be generated with the local imagery basemap. The download will be quicker since no tiles are exported or downloaded. Since the application is not exporting online ArcGIS Online basemaps you will not need to log-in.
+1. Tap on "Generate Offline Map".
+2. You will be prompted to choose whether you wish to download the online basemap or use the "naperville_imagery.tpk" basemap which is already on the device.
```

Comment:
since the user would need to manually add the data to their device if they're running it via Github (the data I think already comes with the app via the App Store?), we could reword this slightly
```suggestion
2. You will be prompted to choose whether you wish to download the online basemap or use the "naperville_imagery.tpk" basemap (see Offline Data section).
```